### PR TITLE
task-67 & task-68

### PR DESCRIPTION
**Pull request by aqsa505**

### CODE REVIEW

**Comment by aqsa505** [comment]
@NComer98 I opened a new pull request with complete form, that one was missing feedback function on form.py

File: cards/views.py [hunk 1]
```diff
@@ -43,8 +45,8 @@ def get_info(request):
             print(banks)
 
             list_of_cards = get_best_cards(groceries, dining_out, gas, travel, everything_else)
-            okay_cards=[]
-            best_cards=[]
+            okay_cards = []
```
Comment (natharrow) [inline cards/views.py:48]: What are okay_cards for?

Comment (aqsa505) [reply cards/views.py:48]: @natharrow I have no idea because i didn't work on this function.

Comment (Masuda50) [reply cards/views.py:48]: Okay cards are the cards being loaded into a list before the user preferences are used to vet them

Comment (Quetourah) [reply cards/views.py:48]: @natharrow this is an array from before. Sometimes when we do PRs changes like white spaces also show up as if we added something new. I'm not sure if I or Masuda created this array., but it holds the initial cards. When the list of cards is generated, we have get an array holding the actual card values. Then we filter through it. This is a function we could have broken down and improved, but that's how it works for now.

Comment (aqsa505) [reply cards/views.py:48]: @Masuda50 Thanks for explaining

**Pull request merged**